src/main/java/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -1,5 +1,30 @@
 [
   {
+    "name": "Data Loss Prevention",
+    "documentation": "https://cloud.google.com/dlp/docs/reference/libraries",
+    "description":"The DLP API enables clients to detect the presence of Personally Identifiable Information (PII) and other privacy-sensitive data in user-supplied, unstructured data streams such as text blocks and images.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/dlp/compute.svg",
+    "clients": [
+      {
+      "name":"Java Client Libraries for the Google Compute Engine API",
+      "compute":"java",
+      "site":"https://cloud.google.com/compute/docs/reference/libraries#client-libraries-install-java",
```

Comment:
https://cloud.google.com/dlp/docs/reference/libraries#client-libraries-install-java

Reply:
These don't need to be checked  manually if anyone will approve https://github.com/GoogleCloudPlatform/appengine-plugins-core/pull/448

Reply:
That won't pick up links for a different component though :-)